links.py

```diff
@@ -0,0 +1,51 @@
+import requests
+import threading
+
+
+def check_url(url):
+    try:
+        status_url = requests.get(url)
+        print("Target: ", url)
+        print("Status: ", status_url.ok, '\n')
+    except:

```

Comment:
Could you please check why it is not a good idea to use bare **except:**?
https://docs.quantifiedcode.com/python-anti-patterns/correctness/no_exception_type_specified.html
https://www.flake8rules.com/rules/E722.html

Reply:
Done